packages/code-editor/src/Panel/Panel.tsx

```diff
@@ -74,6 +75,7 @@ import { PanelProps } from './Panel.types';
 export function Panel({
   baseFontSize: baseFontSizeProp,
   customSecondaryButtons,
+  'data-lgid': dataLgId,
```

Comment:
If a consumer passes a custom data-lgid to `CodeEditor`, it won't propagate down to the panel. So, for example, 

```js
<CodeEditor data-lgid="lg-custom" panel={<Panel/>}>...</CodeEditor>
```

In this case, any component that uses the lgids instantiated in `CodeEditor` will have a data-lgid prefixed with `lg-custom.....`. But Panel and all of its children data-lgids will still be prefixed with`lg-code_editor...`. 

What I've done in instances like this is store the lgids instantiated in the parent component, which is `CodeEditor`, in context and read those values from context in child component. [The `Panel` component in `Code` does this ](https://github.com/mongodb/leafygreen-ui/blob/main/packages/code/src/Panel/Panel.tsx#L32).